backend/tracim_backend/tests/functional/test_session.py

```diff
@@ -340,6 +340,75 @@ def test_api__test_cookie_auth_token__ok__nominal(self):
             )
             assert 'Set-Cookie' in res.headers
 
+    def test_api__test_cookie_auth_token__ok__change_email(self):
```

Comment:
The part after `__ok__` should explain the usecase. Here `change_email` should probably be renamed to something like `change_email_dont_break_cookie`